src/pages/Books.js

```diff
@@ -0,0 +1,12 @@
+import './Books.css';
+
+export default function Books() {
+  return (
+    <main>
+      <div className="books-container">
+        <p>No books added</p>
+        <button type="button">Remove</button>
+      </div>
+    </main>
+  );
```

Comment:
- [ ] Kindly create a new component for adding a book. The component should contain inputs fields for `title` and `author` and a placeholder Add button (no functionality yet). Render the form component here.
- [ ] You are missing a component for rendering a single book. The component should take in book data (`title` and `author`) as a prop and render them. It should also have a placeholder Remove button. Then render that component here also.

Reply:
@TedLivist  I honestly thought that would be in the next project

Reply:
Hi @cwaku not really. As indicated, you are initializing the components at this point in the project.